great-idea/index.html

```diff
@@ -1,23 +1,123 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
 
-  <title>Great Idea!</title>
+    <title>Great Idea!</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
-
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
-</head>
+    <link
+      href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/index.css" />
 
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
   <body>
+    <div class="container">
+        <header>
+            <nav>
+              <a href="services.html">Services</a> <a href="#">Product</a>
+              <a href="#">Vision</a> <a href="#">Features</a> <a href="index.html">About</a>
+              <a href="#">Contact</a>
+            </nav>
+            <div class="logo">
+            <img src="img/logo.png" alt="Great Idea! Company logo." />
+            </div>
+          </header>
```

Comment:
```suggestion

```suggestion
            <header>
          <nav>
              <a href="services.html">Services</a> 
              <a href="#">Product</a>
              <a href="#">Vision</a> 
              <a href="#">Features</a> 
              <a href="index.html">About</a>
              <a href="#">Contact</a>
            
            <div class="logo">
            <img src="img/logo.png" alt="Great Idea! Company logo." />
            </div>
          </nav>
        </header>
```
It's better to line up your html like this, even though the other way will technically work. We want our code to be readable by humans as well as by the browser!

Also, if we include the Great Idea! image inside the nav container, it will be part of the flex-box flow. Maybe you didn't see it on your screen, but at different sizes the logo was getting cut off. This way, the image will be treated as one of the flex-box elements. I've also made some css suggestions that will make the nav look pretty and be responsive.